tests/bears/BearTest.py

```diff
@@ -463,6 +473,30 @@ def test_bear_with_specific_language(self):
         self.assertEqual(str(result), 'Hypertext Markup Language 5.1')
         self.check_message(LOG_LEVEL.DEBUG)
 
+    @patch('pdb.Pdb.do_continue')
+    def test_do_continue(self, do_continue):
+        arg = {}
+        self.assertEqual(dbg.do_continue(arg), 1)
+
+    @patch('coalib.bears.Bear.dbg.runcall', side_effect=((1, 2), 3, 4))
+    def test_debug_mode_func_with_return(self, runcall):
+        section = Section('name')
+        my_bear = TestOneBear(section, self.queue, debugger=True)
+        args = {}
+        kwargs = {}
```

Comment:
please read my comments on the old PR.

Reply:
I know you said arg is a tuple and you said -->`Do you know you are passing empty dictionary` but on the `gitter` channel :P, actually these `arg and `kwargs` will not get used as I am using `mock.patch`.